### DISCUSSION TITLE
How can I open the menu?

### DISCUSSION THREAD

**Discussion by Quimizz** [maruohon/litematica#497] [Q&A] [answered]
I accidentally put the easy place hotkey as my menu hotkey and now I can't open it to change it. I deleted the mod from my folder and installed it again but it didn't reset my settings.

**Comment by maruohon** [comment, marked as answer]
You can open the config menu via the Mod Menu mod list if you have Mod Menu installed, or edit the config file at `.minecraft/config/litematica.json`.

**Reply by Quimizz** [reply]
Thanks a lot!
